aemedge/blocks/popular-blogs/popular-blogs.css

```diff
@@ -1,12 +1,181 @@
-.section.popular-blogs-container
-{
-    background: #f2f2f2;
+.section.popular-blogs-container {
     margin: 0 auto;
     width: 100%;
     padding: 37px 0;
+    background: #f2f2f2;
```

Comment:
Are any of these colors used more than once? Or check if we already have variables...